Rakefile

```diff
@@ -1,12 +1,12 @@
-# encoding: UTF-8
-
 require 'bundler/gem_tasks'
 require 'rake/testtask'
 require 'rdoc/task'
 
 desc 'Run tests for InheritedResources.'
 Rake::TestTask.new(:test) do |t|
-  t.pattern = 'test/**/*_test.rb'
+  t.libs << "test"
+  t.libs << "lib"
+  t.test_files = FileList['test/**/*_test.rb']
```

Comment:
I'm unsure of the difference between `test_files` and `pattern`, but if this is what the default gem skeleton uses, I'm good with it! Adding `test` and `lib` to the LOAD_PATH seems definitely a good idea :+1:.

Reply:
Yes neither am I. I just went off of past experience making a gem where I noticed regardless of the file location, I could just simply have `require 'test_helper'` which I liked. I figured this had to do with it which it seems too. Thanks!

Reply:
Actually, I think I have a reason to prefer `pattern` over `test_files`. RSpec also uses a [`pattern` option](https://github.com/rspec/rspec-core/blob/9d1d0d6aeb8dc87ff4e584cdcf9e8b1a234da2ae/lib/rspec/core/rake_task.rb#L35) for the exact same thing, so by using it here too, a new developer doesn't need to learn a new API, no matter what test framework is in use. If you agree, we can change it back and instead propose to change the template in bundler :)

Reply:
@deivid-rodriguez sure that sounds good to me! 😄 I went ahead and made the change. I'll merge this first once the build passes. Thanks! 👏🏻